src/util.h

```diff
@@ -764,6 +766,31 @@ class PersistentToLocal {
   }
 };
 
+class CPUInfo {
+ public:
+  CPUInfo() {
+    if (uv_cpu_info(&info_, &count_) != 0) {
+      info_ = nullptr;
+      count_ = 0;
+    }
+  }
+  ~CPUInfo() {
+    if (info_ != nullptr)
+      uv_free_cpu_info(info_, count_);
+  }
+  int count() const { return count_; }
+  operator bool() const {
```

Comment:
Nit: I think we usually try to put empty lines in between methods. Could you also run `make format-cpp` for consistency?

Reply:
Sigh...

```
C:\Users\jasne\Projects\node>vcbuild format-cpp
Error: invalid command line option `format-cpp`.
```

Yeah, when I update this I'll switch over to the linux box and tweak the formatting.